misk/build.gradle

```diff
@@ -83,6 +83,7 @@ dependencies {
   compile dep.prometheusClient
   compile dep.prometheusHotspot
   compile dep.jnrUnixsocket
+  compile dep.miskWeb
```

Comment:
Reverted this change because `misk-web` artifacts aren't publicly available for Travis builds yet. https://github.com/square/misk-web/issues/29